modules/alloy/src/test/ts/atomic/ui/slider/SliderModelTest.ts

```
@@ -59,8 +56,10 @@ UnitTest.test('Atomic Test: ui.slider.SliderModelTest', () => {
   // Reducing never goes beyond min-1
   fc.assert(fc.property(arbData, (data) => {
     const newValue = SliderModel.reduceBy(data.value, data.min, data.max, data.stepSize);
+    const assertion = newValue <= data.value && newValue >= data.min - 1;
+    if (!assertion) debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)